src/apps/allbridge/api.ts

```diff
@@ -98,7 +99,41 @@ export async function getAllbridgeTokenInfo({
   ) as AllbridgeApiResponse
   if (!allbridgeTokensInfoResponse) {
     allbridgeTokensInfoResponse = await got
-      .get('https://core.api.allbridgecoreapi.net/token-info')
+      .get('https://core.api.allbridgecoreapi.net/token-info', {
+        hooks: {
+          beforeRequest: [
+            (options) => {
+              logger.debug('allbridge: request details', {
+                url: options.url.toString(),
+                headers: options.headers,
+                method: options.method,
+              })
+            },
+          ],
+          beforeError: [
+            (error) => {
+              logger.error('allbridge: request error', {
+                statusCode: error.response?.statusCode,
+                statusMessage: error.response?.statusMessage,
+                responseHeaders: error.response?.headers,
+                responseBody: error.response?.body,
+                url: error.options?.url?.toString(),
+                requestHeaders: error.options?.headers,
+              })
+              return error
+            },
+          ],
+          afterResponse: [
+            (response) => {
+              logger.debug('allbridge: response received', {
+                statusCode: response.statusCode,
+                headers: response.headers,
+              })
+              return response
+            },
+          ],
+        },
+      })
```

Comment:
We already capture and log all this.
Example: https://console.cloud.google.com/logs/query;cursorTimestamp=2025-02-25T11:23:36.625Z;duration=P1D;query=resource.type%20%3D%20%22cloud_run_revision%22%0Aresource.labels.service_name%20%3D%20%22hooks-api%22%0Aresource.labels.location%20%3D%20%22us-central1%22%0A%20severity%3E%3DDEFAULT%0Aseverity%3DERROR%0Atimestamp%3D%222025-02-25T11:23:36.625Z%22%0AinsertId%3D%2267bda838000989af07ce8051%22;storageScope=project?project=celo-mobile-mainnet